packages/vue/src/examples/pages/checkout/_internal/Shipping.vue

```diff
@@ -152,13 +173,13 @@ export default {
     SfRadio
   },
   props: {
-    order: {
-      type: Object,
-      default: () => ({})
-    },
     shippingMethods: {
       type: Array,
       default: () => []
+    },
+    value: {
+      type: Object,
+      default: () => ({})
     }
   },
   data() {
```

Comment:
@martaradziszewska maybe we can improve `countries: []` and use this API ? https://restcountries.eu/rest/v2/all?fields=name